incubator/vault/values.yaml

```diff
@@ -26,6 +26,10 @@ consulAgent:
 service:
   name: vault
   type: ClusterIP
+  # type: LoadBalancer
```

Comment:
Please make it an empty list by default.

```yaml
  loadBalancerSourceRanges: []
  #   - 10.0.0.0/8
  #   - 130.211.204.2/32
```

Reply:
Changes made!